Taeryoung/app.js

```diff
@@ -0,0 +1,40 @@
+
+const express = require ("express");
+const cors = require ("cors");
+const morgan = require ("morgan");
+const { DataSource } = require('typeorm');
+const dotenv = require("dotenv");
+dotenv.config()
+
+const app = express()
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan('combined'));
+
+const myDataSource = new DataSource({
+    type: process.env.TYPEORM_CONNECTION,
+    host: process.env.TYPEORM_HOST,
+    port: process.env.TYPEORM_PORT,
+    username: process.env.TYPEORM_USERNAME,
+    password: process.env.TYPEORM_PASSWORD,
+    database: process.env.TYPEORM_DATABASE
+});
+
+myDataSource.initialize()
+    .then(() => {
+        console.log("Data Source has been initialized!")
+    })
+
+
+    app.get('/ping', cors(), function (req, res, next) {
```

Comment:
```suggestion
    app.get('/ping', function (req, res, next) {
```
위에서 `app.use(cors())` 코드를 작성함으로써 cors 관련 처리가 이미 되어있는 것 같아요.

추가로 express에서 활용하는 middleware 패턴은 request가 들어와 API가 호출될 때까지 중간에 거쳐가는 다리라고 생각하시면 됩니다.